master/getting-started/kubernetes/tutorials/stars-policy/policies/default-deny-client.yaml

```diff
@@ -0,0 +1,7 @@
+kind: NetworkPolicy
+apiVersion: extensions/v1beta1
+metadata:
+  name: default-deny
+  namespace: client
```

Comment:
This is fine, but alternatively we could not specify a Namespace and just use the `-n` flag to decide which ns we put it in.